apps/newsletters-ui/src/app/components/NewslettersTable.tsx

```diff
@@ -1,12 +1,12 @@
 import { useMemo } from 'react';
 import { Link } from 'react-router-dom';
 import type { Column } from 'react-table';
-import type { Newsletter } from '@newsletters-nx/newsletters-data-client';
+import type { LegacyNewsletter } from '@newsletters-nx/newsletters-data-client';
 import { formatCellBoolean } from './Cell';
 import { Table } from './Table';
 
 interface Props {
-	newsletters: Newsletter[];
+	newsletters: LegacyNewsletter[];
 }
 
 export const NewslettersTable = ({ newsletters }: Props) => {
```

Comment:
This this file specifically uses LegacyNewsletter in the Props, should it be renamed LegacyNewslettersTable?

Reply:
I think that's out of scope for this PR - it was just meant to be renaming the existing type rather than creating any new ones or changing how anything works.